src/Extensions/Diagnostics.HealthChecks/Internal/RestClientWrapper.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.ServiceFabric.Client;
+using Microsoft.ServiceFabric.Client.Http;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	internal class RestClientWrapper
+	{
+		private ServiceFabricHttpClient m_client;
+
+		private readonly Uri m_clusterEndpoint;
+
+		private bool m_ready;
+
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
Suppressing warnings should be the last resort, I think in this case it would be better to make field `m_client` nullable (with type `ServiceFabricHttpClient?`) and check it for null in `Get` method. Then you also won't need `m_ready` field.

Reply:
OK, done